strategy/plan2020.md

```diff
@@ -0,0 +1,208 @@
+  that we absolutely *should* be engaged?
+- **Stop:** What activities are we currently engaged in that we
+  should *stop* doing immediately.
+- **Continue:** What activities are we currently engaged in that
+  we should endeavor to continue.
+
+After 30 minutes of brainstorming ideas in each category above,
+each participating member of the Xdev Team was given 5 points to
+distribute across activites listed in the *Start* and *Stop*
+categories, essentially given everyone on the Team the ability
+to vote for what *changes* should be taken in the future of our
+work.  The results of this exercise are detailed below.
+
+## Strategic Activities
+
+In our own terminology, the Xdev Team's purpose/mission can be
+categorized into two different *Campaigns*: *Workflows* and
+*Outreach, Education & Community Building*.  As a team, the work
+we do specifically engages in short-term *Projects* targeting
+desired features or outcomes related to these Campaigns.  In
+the remainder of this document, we use the term *Activity* to
+refer to a concept that can be broken down into 1 or more
+Projects targeting a single Campaign.
+
+### Continuing Activities
+
+These are Activities in which the Xdev Team is currently engaged
+and are expected to continue.
+
+1. Binder on HPC Investigation
+   - Describe the general desired outcome
+     - e.g., Enable the ability to let any (authorized) user launch
+       on Cheyenne/Casper a Jupyter Notebook with the fixed
+       environment defined by the author
+   - Enumerate the ways that this general desired outcome can be achieved
+   - Sketch rough project plans for each enumerated way
+   - Prioritize enumerated project plans based on practical factors:
+     - How easy is it to implement?
+     - How easy is it to test?
+     - How long will it take?
+     - What risk factors are involved?
+   - Develop an implementation plan, following enumerated paths until
+     a workable solution is found
+2. Cataloging GLADE
+   - What datasets are available on GLADE?
+   - Who owns or manages those datasets?
+   - How can we *easily* generate intake catalogs for those datasets?
+   - Train data managers to create catalogs for available datasets
+   - Create tools to help data managers create catalogs easily
+   - How can we aggregate dataset catalogs into a single, searchable catalog?
+3. Notebook Examples
+   - Identify good example notebooks that:
+     - exist already (and can be spruced up for sharing), and
+     - could be written (e.g., examples that could be useful to others)
+   - Add notebooks to the Pangeo Gallery (gallery.pangeo.io)
+   - Identify examples that make good blog posts, too!
+4. Python Tutorials
+   - Take tutorials virtual until COVID is not a risk
+   - Grow online, self-paced content
+
+### New Activities
+
+These are Activities in which the Xdev Team plan to engage in the
+next year (July 2020 - June 2021).  Each individual activities shows
```

Comment:
activities -> activity

Reply:
Good catch!  Will fix.

Reply:
Actually, you are looking at an old revision.  You might need to refresh.  That error has been fixed already.